pages/api/waitlist/index.ts

```diff
@@ -27,19 +26,19 @@ export default async function handle(req: NextApiRequest, res: NextApiResponse):
     switch (req.method) {
         // Retrieving a waitlist by passing in the classId and parentId.
         case "GET": {
-            const input = req.body as WaitlistInput;
+            const { classId } = req.body;
```

Comment:
Maybe we move this up to remove code redundancy? Seems like it's used in all cases.